app/views/yablog/admin/shared/_sidebar.html.haml

```diff
@@ -0,0 +1,18 @@
+.sidebar-nav.navbar-collapse
+  %ul#side-menu.nav
+    / %li.sidebar-search
```

Comment:
it would make sense to leave some comments why exactly it was commented out to not bring garbage